lib/gemfury/command/app.rb

```diff
@@ -295,6 +315,59 @@ def push_files(command, gem_paths)
     end
   end
 
+  C50K = 50000
+
+  class ProgressIO < SimpleDelegator
+    attr_reader :content_type, :original_filename, :local_path, :io
```

Comment:
You can probably get rid of `@io` and `attr_reader :io`, and use SimpleDelegator's `__getob__` instead. You could use `alias_method :io, :__getobj__`, if you want to keep the external API here.

Reply:
Ahh good catch. I don't think i need this externally exposed. So will remove instead.